templates/consumption_moment/index.html.twig

```diff
@@ -4,6 +4,14 @@
     <div class="flex flex-col w-1/2 mx-auto pt-8">
         <div class="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
             <div class="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+                <div class="flex pb-4">
```

Comment:
Deze code is niet nodig om toe te voegen. Onderstaand regel 9 t/m ?? is dit al beschikbaar, enkel wanneer er een arduino bekend is van de user.